command.go

```diff
@@ -102,6 +102,19 @@ type CommandFunc struct {
 	// See Command for details about the accepted signatures.
 	Func interface{}
 
+	// An optional usage string for this function. If set, then this replaces
+	// the default one that shows the types (but not names) of arguments.
+	Usage string
+
+	// Set of options to not set from the environment
+	// more user-friendly-syntax
+	IgnoreEnvOptions []string
+
+	// Set of options to not set from the environment
+	// This is to convert IgnoreEnvOptions field to a map
+	// for efficient lookups
```

Comment:
How do these interact if a user specifies both?

Reply:
Ah. I guess my comment was just from a user perspective, is one preferred? If you specify both, is one ignored? It's not obvious from the docs what will happen.

Reply:
Sorry, I miss your comment and was on PTO most of the last week. Let me update the comment and then merge it! 
Thanks for the reviewing